connected-car-lagom-impl/src/main/resources/application.conf

```diff
@@ -23,75 +23,60 @@ akka.actor.serialization-bindings {
 }
 
 akka {
-
   loglevel = DEBUG
 
-  discovery {
-    method = marathon-api
-
-    marathon-api {
-      class = akka.discovery.marathon.MarathonApiSimpleServiceDiscovery
-
-      # URL for getting list of apps from Marathon. Verified on OSS DC/OS 1.8, 1.9
-      app-api-url = "http://marathon.mesos:8080/v2/apps"
-
-      # The name of the akka management port - this cannot have underscores or dashes (env var name)
-      app-port-name = "akkamgmthttp"
-
-      # Used to find other apps running by Marathon. This will be passed as the label query string parameter
-      # to the apps-api-url defined above.
-      # `%s` will be replaced with the configured effective name, which defaults to the actor system name
-      app-label-query = "ACTOR_SYSTEM_NAME==%s"
-    }
-  }
-
```

Comment:
Removed this as it's simply a copy of the defaults. I think our example should be simplified too.